src/hotspot/share/opto/escape.hpp

```diff
@@ -473,7 +481,7 @@ class ConnectionGraph: public ArenaObj {
   const TypeInt* optimize_ptr_compare(Node* n);
 
   // Returns unique corresponding java object or NULL.
-  JavaObjectNode* unique_java_object(Node *n);
+  JavaObjectNode* unique_java_object(Node *n) const;
```

Comment:
Was adding `const` required by your changes, or is it just cleanup?

Reply:
Not a requirement.